src/AltinnCore/Common/Models/DeploymentResponse.cs

```diff
@@ -0,0 +1,28 @@
+namespace Common.Models
+{
+    /// <summary>
+    /// Model for deployment responses
+    /// </summary>
+    public class DeploymentResponse
+    {
+        /// <summary>
+        /// Success status
+        /// </summary>
+        /// <value></value>
```

Comment:
<value></value> can be removed since they give nothing to the code

Reply:
Fixed.